package.json

```diff
@@ -2,17 +2,14 @@
   "name": "redux-observable",
   "version": "1.0.0-alpha.1",
   "description": "RxJS based middleware for Redux. Compose and cancel async actions and more.",
-  "module": "lib/esm/index.js",
-  "main": "lib/cjs/index.js",
+  "main": "./lib/redux-observable.cjs.js",
+  "module": "./lib/redux-observable.esm.js",
+  "browser": "./dist/redux-observable.umd.js",
```

Comment:
My initial reaction is that I'd prefer not to change these locations. Lmk if there's a good argument for it? Also curious: where the rest of the flat files, other than the entry points, live if everything is built into the same directory?

Reply:
Change lib to dist?

Reply:
Change `lib` to `dist`?